lib/pbench/server/database/alembic.ini

```diff
@@ -7,8 +7,14 @@ script_location = alembic
 # template used to generate migration files
 # file_template = %%(rev)s_%%(slug)s
 
-# timezone to use when rendering the date
-# within the migration file as well as the filename.
+# sys.path path, will be prepended to sys.path if present.
+# defaults to the current working directory.
+prepend_sys_path = .
```

Comment:
s/sys.path/prepend_sys_path/ on the first match.

As long as you're there, I kinda hate the word "prepend".  How about "prefix" or "affix" or "insert at the beginning of"....

Reply:
This is all autogenerated code so I didn't touch any of this after running `alembic init alembic` again, only to add the `Database.Base.metadat`.

Reply:
If you still think we should change this after reviewing the draft PR #3180 which shows how this file is different from the autogenerated one, let me know.